src/js/burials/config/form.js

```diff
@@ -67,6 +79,66 @@ const formConfig = {
           }
         }
       }
+    },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
```

Comment:
I could probably use the common applicantInformation page for this, but I'd have to add a couple more configurable bits and it seemed like it was stretching the purpose of that shared page too much.